doc/source/rest.j2

```diff
@@ -187,7 +222,7 @@ By default, new |measures| can only be processed if they have timestamps in the
 future or part of the last aggregation period. The last aggregation period size
 is based on the largest |granularity| defined in the |archive policy|
 definition. To allow processing |measures| that are older than the period, the
-`back_window` parameter can be used to set the number of coarsest periods to
+`back_window` parameter can be used to set the number of historical periods to
```

Comment:
Except you don't need the number of historical period.
We need to change that unit anyway. Logging a big.

Reply:
i'm not entirely sure what you mean by

> We need to change that unit anyway. Logging a big.

you want me to change something in this patch?

Reply:
s/big/bug/

I opened #459 to track that. I don't think your change here makes sense.
That does not mean I think the original text is easy to understand. But this line change remove information.
So it makes from *hard* to *impossible* to understand the parameter. :)

Reply:
done